guidelines/terms/21/single-pointer.html

```diff
@@ -1,7 +1,5 @@
 <dt><dfn>single pointer</dfn></dt>
 <dd>
-   	
-   <p>pointer input that operates with one point of contact with the screen, including single taps and clicks, double-taps and clicks, long presses, and path-based gestures</p>
-  				
+   <p>an input that only targets a single point on the page/screen at a time – such as a mouse, stylus, or single finger on a touch screen – as opposed to multi-pointer/multi-touch scenarios</p>
```

Comment:
Combining one of @alastc 's suggestions with a suggested note to disambiguate make it clear whether stuff like double-clicks counts, based on @scottaohara 's observation that this is a recurring point of confusion for readers.

```suggestion 
   <p>an input that only targets a single point on the page/screen at a time</p>

   <p class="example">A mouse, stylus, or single finger on a touch screen. A multi-touch interface requiring the use of multiple fingers simultaneously would not be "single pointer".</p>
   
   <p class="note">Functionality might require using a single pointer multiple times in sequence; as long as the input only targets a single point at a time, this still represents using a single pointer. For example, double-clicking, long-pressing, and dragging a slider are common operations performed using a single pointer.</p>
```